Replaced gh-actions-cache with git cache subcommand (#28850)
github/docs: content/actions/using-workflows/caching-dependencies-to-speed-up-workflows.md

- Using the {% data variables.product.prodname_dotcom %} web interface, as shown below.
- Using the REST API. For more information, see the "[AUTOTITLE](/rest/actions/cache)" REST API documentation.

> - Installing the `gh cache` subcommand to manage your caches from the command line. For more information, see the [GitHub CLI documentation](https://cli.github.com/manual/gh_cache). Note that you should ensure that you have version 2.32.0 or higher of the CLI installed if you are doing this manually.

Reviewer: ```suggestion
- Installing the `gh cache` subcommand to manage your caches from the command line. For more information, see the [GitHub CLI documentation](https://cli.github.com/manual/gh_cache). 

    {% note %}
    
    **Note:** If you are doing this manually, ensure you have version 2.32.0 or higher of the CLI installed.
    
    {% endnote %}
```
Hey @dheerajcl, this is a great update to the docs! 💛 I am making one small formatting change to add the note in a blue callout box. This isn't something we'd expect contributors to know how to do, or even really know is an option on their first contribution, so you did great writing exactly what you did. We do have a [style guide](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide) with this information if you plan on contributing more, but we wouldn't expect anyone to memorize it because it's _a lot_. I'm going to commit this update, and then I'll get your PR merged! Thank you again for helping us improve the docs! 🚀
